data/data-pipeline/data_pipeline/score/utils.py

```diff
@@ -0,0 +1,56 @@
+"""Utilities to help generate the score."""
+import pandas as pd
+import geopandas as gpd
+import data_pipeline.score.field_names as field_names
+
+# XXX: @jorge I am torn about the coupling that importing from
```

Comment:
Cannot speak to best practice here, but I think what you have here is good.

Reply:
@mattbowen-usds: I just realized the existence of `data_pipeline.etl.sources.geo_utils`. It looks like this was done in a prior PR. Was there a reason to put the functions there instead of adding these to `data_pipeline.utils`? Was there circular imports issues?

Reply:
I have referenced this here: https://github.com/usds/justice40-tool/issues/1839